src/tests/push-undelete-config-dev/expected-stdout

```diff
@@ -0,0 +1,5 @@
+Plan for "push" operation:
+	+  C dev/extractor/keboola.ex-db-mysql/undeleted
```

Comment:
Tento config v projekte existuje ako `[TO DELETE]`, pre CLI ako-keby neexistoval, takze ho vytvori .... ale realne ho iba aktualizuje a zmaze z mena `[TO DELETE]`, .... trocha komplikovane :) ale implementovat to poriadne v API sa asi nestihalo.